src/pages/Profile/Profile.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../styles/variables" as *;
+
+.profile {
+  font-family: "poppins";
+  font-size: 1.125rem;
+  background-color: $off-white-color;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 3rem 0 0 0;
+  height: 100vh;
+  &__img {
+    height: 9rem;
+    width: 9rem;
```

Comment:
Consider using em here instead of rem as per the styling guidelines, if there's an issue let me know